misc/net.minetest.minetest.appdata.xml

```diff
@@ -11,26 +11,44 @@
 	</content_rating>
 	<name>Minetest</name>
 	<summary>Multiplayer infinite-world block sandbox game</summary>
+	<summary xml:lang="de">Mehrspieler-Sandkastenspiel mit unendlichen Blockwelten</summary>
 	<description>
 		<p>
 			Minetest is an infinite-world block sandbox game and game engine.
+		</p><p xml:lang="de">
+			Minetest ist ein Sandkastenspiel und eine Spielengine mit unendlichen Welten.
 		</p><p>
 			Players can create and destroy various types of blocks in a
 			three-dimensional open world. This allows forming structures in
 			every possible creation, on multiplayer servers or in singleplayer.
+		</p><p xml:lang="de">
+			Spieler können in einer offenen 3D-Welt viele verschiedene Arten von
+			Blöcken platzieren und abbauen. Dies erlaubt das Bauen von allen möglichen
```

Comment:
> Dies erlaubt das Bauen von allen möglichen Strukturen

Should probably be worded differently (das Bauen von allerlei/verschiedensten/vielfältigen/... Strukturen), although I fail to grasp the intention of the English version ("in every possible creation").

Reply:
I vote for "vielfältigen", it's a great word